app/tasks/email.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import \
+    unicode_literals, \
+    absolute_import
+
+from . import celery
+import boto3
+import botocore.exceptions
+import textwrap
+import sys
+from flask import current_app
+from flask._compat import string_types
+from dmutils.email import hash_email, to_bytes, to_text, EmailError
+from string import Template
+from os import getenv
+
+
+@celery.task
+def send_email(to_email_addresses, email_body, subject, from_email, from_name, reply_to=None):
```

Comment:
is this essentially of the existing send_email code?

Reply:
Yes it's essentially the same as the utils function of the same name now that the utils function also checks the AWS env vars. We could probably just wrap dmutils.email.send_email?

Reply:
I'm fine not - we should start to transition from using utils wherever possible